src/box/schema.cc

```diff
@@ -288,7 +288,7 @@ schema_init(void)
 	key_parts[1].fieldno = 1; /* child space */
 	key_parts[1].type = FIELD_TYPE_UNSIGNED;
 	sc_space_new(BOX_FK_CONSTRAINT_ID, "_fk_constraint", key_parts, 2,
-		     &on_replace_fk_constraint);
+		     NULL);
 
```

Comment:
Why don't you simply remove this system space definition now? It doesn't seem to be required. Also, why not drop the space in the upgrade script?

Reply:
Do you know a way to drop system space with primary index defined? Method `drop()` is not working:
```
tarantool> box.space._fk_constraint:drop()
---
- error: Can't drop the primary key in a system space, space '_fk_constraint'
...
```
I think we can add an internal function to remove system space. Should I do it in this PR?

Reply:
> Do you know a way to drop system space with primary index defined?

I guess you could update the check to allow removal of these two old spaces. Their ids should be removed too (replaced with 'RESERVED' values).

> I think we can add an internal function to drop system space.

A space is removed by deleting a record from a system space. I don't imagine how we could circumvent that.

> Should I do it in this PR?

Without it, this PR shouldn't be called "**Completely** drop SQL constraints" :)

Reply:
I will create a new PR where I'll drop these system spaces. For now, I've renamed the PR and edited the commit messages.